pkg/build/api/v1/types.go

```diff
@@ -402,6 +402,11 @@ type SourceBuildStrategy struct {
 
 	// ForcePull describes if the builder should pull the images from registry prior to building.
 	ForcePull bool `json:"forcePull,omitempty"`
+
+	// disableImplicitBuild prevents source-to-image from performing a Docker build operation
```

Comment:
(and frankly, a user who wants to protect themselves from ONBUILD commands in a builder image they chose, probably *is* a good justification for making this available to users)